src/components/Col/Col.spec.js

```diff
@@ -39,4 +39,9 @@ describe('Create element', () => {
     expect(wrapper.dive().type()).toEqual('span');
     expect(wrapper.dive().type()).not.toEqual('div');
   });
+
+  test('generates custom with custom display css property', () => {
+    const tree = renderer.create(<Col display={'flex'} />).toJSON();
+    expect(tree).toHaveStyleRule('display', '-webkit-box');
```

Comment:
I don't understand why this assertion in jest returns like first value `-webkit-box`. if I put `flex` the test fail.